item-spec/examples/itemcollection-sample-minimal.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I feel like this is too minimalist. I think this could be removed without loosing any valuable information.

Reply:
I think it's important to specify the minimal case.  I think it makes clear what is optional.